coverage_assessment.R

```diff
@@ -41,7 +41,118 @@ nut <- get_data()
 
 # Problem 3: Prevalence of acute undernutrition/malnutrition ----
 
+library(dplyr)
+
+## Load functions in R directory ----
+for (f in list.files("R", full.names = TRUE)) source (f)
+
+
+# Read data ----
+nut <- get_data()
+
+
+####MAM
+
+with(nut, sum(muac >= 11.5 & muac <12.5  & age >=6 & age <=59 & oedema == 2, na.rm= TRUE))
+
+##number of moderate acute malnourished (MAM) cases children 6-59 months old 
+
+Male_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 & age >=6 & age <=59 & oedema == 2 & sex == 1 , na.rm= TRUE ))
+
+##assigning moderate acute malnourished (MAM) male 6-59 months old to Male_MAM
+
+Female_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 & age >=6 & age <=59 & oedema == 2 & sex == 2 , na.rm= TRUE ))
+
+##assigning  moderate acute malnourished (MAM) female 6-59 months old to Female_MAM
+
+UM_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 & age >=6 & age <=59 & oedema == 2 & county == "Urban Montserrado" , na.rm= TRUE))
+
+##assigning moderate acute malnourished (MAM) cases children 6-59 months old  in Urban Montserrado to UM_MAM
+
+GB_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 &age >=6 & age <=59 & oedema == 2 & county != "Urban Montserrado" , na.rm= TRUE))
```

Comment:
In terms of the grouping by location, you count the MAM cases for Grand Bassa by doing a negation of Urban Montserrado. Again, this is correct for our dataset but in other datasets where let's say there is a third county or let's say if a county was unknown and the data was recorded as "County unknown" rather than NA, then this row/s of data will be counted as `county != "Urban Montserrado"` rather than not being counted at all. So, a more universal code will be to use `county == "Grand Bassa". For our dataset, this doesn't matter so this is still ok. You don't have to change it.

Reply:
Understood. Thanks!